int64 req
---
float64[] res